packages/vsphere/data_stream/network/sample_event.json

```
@@ -0,0 +1,76 @@
+{
+      "@timestamp": "2024-09-22T21:01:42.635Z",
+      "agent": {
+        "ephemeral_id": "b4116483-d4c6-4860-b93d-f0d8091cc838",
+        "id": "ff0ab35a-1abe-47a1-aee7-6d70362e4335",
+        "name": "docker-fleet-agent",
+        "type": "metricbeat",
+        "version": "8.15.2"
+      },
+      "data_stream": {
+        "dataset": "vsphere.network",
+        "namespace": "default",
+        "type": "metrics"
+      },
+      "ecs": {
+        "version": "8.11.0"
+      },
+      "elastic_agent": {
+        "id": "ff0ab35a-1abe-47a1-aee7-6d70362e4335",
+        "snapshot": true,
+        "version": "8.15.2"
+      },
+      "event": {
+        "agent_id_status": "verified",
+        "dataset": "vsphere.network",
+        "duration": 10826519,
+        "ingested": "2024-09-22T21:01:43Z",
+        "module": "vsphere"
+      },
+      "host": {
+        "architecture": "x86_64",
+        "containerized": true,
+        "hostname": "elastic-agent-23128",
+        "id": "57723763cd1b4ff48e54a505de4ebe6c",
+        "ip": [
+            "192.168.240.2",
+            "192.168.255.5"
+        ],
+        "mac": [
+            "02-42-C0-A8-F0-02",
+            "02-42-C0-A8-FF-05"
+        ],
+        "name": "elastic-agent-23128",
+        "os": {
+            "codename": "focal",
+            "family": "debian",
+            "kernel": "3.10.0-1160.118.1.el7.x86_64",
+            "name": "Ubuntu",
+            "platform": "ubuntu",
+            "type": "linux",
+            "version": "20.04.6 LTS (Focal Fossa)"
+        }
+    },
+      "metricset": {
+        "name": "network",
+        "period": 20000
```

This formatting seems wrong, can you check once?
```suggestion
    },
    "metricset": {
      "name": "network",
      "period": 20000
```

---

Should work fine but re-formatted it just in case.